lib/ansible/modules/monitoring/uptimerobot.py

```diff
@@ -79,7 +79,7 @@ def checkID(module, params):
     data = urlencode(params)
     full_uri = API_BASE + API_ACTIONS['status'] + data
     req, info = fetch_url(module, full_uri)
-    result = req.read()
+    result = req.read().decode("utf-8")
```

Comment:
This should utilize `ansible.module_utils._text.to_native` as opposed to using `decode` here.

Reply:
done!

Reply:
@sivel Hi, could you please look at it again? It would be nice to get this merged :)

Reply:
@koenw Based on https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md#for-pull-request-submitters I see that in order to proceed, you as a pull request author should use `ready_for_review` command. Could you do that, please? (I can't as I'm not neither an author nor maintainer).